vagrantconfig.yaml

```diff
@@ -9,4 +9,4 @@ memory: 256
 debug_mode: false
 
 # set this to "gui" if vagrant cannot connect
-boot_mode: none
+boot_mode: gui
```

Comment:
Just to clarify, it needs to be ``none`` with a lowercase "n". That works for Fjord, so I'm not sure why you had problems. What was the full error you got?